README.md

```diff
@@ -54,3 +54,190 @@ For a simple, gentler, introduction to Lagom, have a look at the [Hello World gi
 - Getting help
 
 If you have any troubles and need help, feel free to ask in the [Gitter channel](https://gitter.im/lagom/lagom)
+
+## Running: Kubernetes
+
+This project uses [Lightbend's Platform Tooling](https://s3-us-west-2.amazonaws.com/rp-tooling-temp-docs/home.html) to
+simplify deployment to [Kubernetes](https://kubernetes.io/).
+
+### Setup
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+* [Minikube](https://github.com/kubernetes/minikube) v0.25.0 or later (Verify with `minikube version`)
+* [Helm](https://github.com/kubernetes/helm)
+* [reactive-cli](https://github.com/lightbend/reactive-cli) 0.9.0 or later (Verify with `rp version`)
+
+
+### Deployment Workflow
+
+> Note that this is an alternative to the Operations workflow documented below.
+
+The following workflow is well suited for development purposes. It will build Docker images for each subproject, generate Kubernetes
+YAML, and deploy them into your Minikube using `kubectl`. It also installs the [Reactive Sandbox](https://github.com/lightbend/reactive-sandbox)
+in your Minikube, providing the required dependencies for Kafka, Cassandra, and Elasticsearch.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+`minikube start --memory 6000`
+
+`sbt "deploy minikube"`
```

Comment:
I'm getting this error:

```
java.lang.RuntimeException: reactive-cli is not installed [132]
	at scala.sys.package$.error(package.scala:27)
	at com.lightbend.rp.sbtreactiveapp.cmd.package$.runSuccess(package.scala:63)
	at com.lightbend.rp.sbtreactiveapp.cmd.rp$.assert(rp.scala:24)
	at com.lightbend.rp.sbtreactiveapp.BasicApp$$anonfun$projectSettings$37$$anonfun$apply$15$$anonfun$apply$16.apply(App.scala:260)
	at com.lightbend.rp.sbtreactiveapp.BasicApp$$anonfun$projectSettings$37$$anonfun$apply$15$$anonfun$apply$16.apply(App.scala:231)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

I have `reactive-cli` installed, but the symlink created by homebrew is `rp` instead of `reactive-cli`.

Reply:
> It succeeded the second time. It's possible that the first time overlapped with a network glitch that I had, but since it's all running locally I wouldn't have expected it to affect anything. In any case, it's hard to tell from the output what the actual problem is or how to troubleshoot it.

It has to pull down reactive sandbox during that step from Bintray, which is about 200MB, so if there was a network glitch that would indeed cause the problem. 

I think we could at least guide the user to kubectl in that event, I've filed https://github.com/lightbend/sbt-reactive-app/issues/91 to track that.

--------------------------------

> When I run rp version, I get: [1] 37496 illegal hardware instruction rp version.

Hrm - I'm curious what CPU/mac model and macOS version?